R/loadFromHumann.R

```diff
@@ -0,0 +1,189 @@
+#' Import HUMAnN results to \code{TreeSummarizedExperiment}
+#
+#' @param file a single \code{character} value defining the file
+#' path of the HUMAnN file. The file must be in merged HUMAnN format.
+#'
+#' @param colData a DataFrame-like object that includes sample names in
+#' rownames, or a single \code{character} value defining the file
+#' path of the sample metadata file. The file must be in \code{tsv} format
+#' (default: \code{colData = NULL}).
+#' 
+#' @param ... additional arguments:
+#' \itemize{
+#'   \item{\code{assay.type}:} {A single character value for naming 
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{assay}} 
+#'   (default: \code{assay.type = "counts"})}
+#'   \item{\code{removeTaxaPrefixes}:} {\code{TRUE} or \code{FALSE}: Should
+#'     taxonomic prefixes be removed? (default:
+#'     \code{removeTaxaPrefixes = FALSE})}
+#' }
+#'
+#' @details
+#' Import HUMAnN (version 3.0) results of pathways or gene families. Input must
```

Comment:
Import HUMAnN (currently version 3.0 supported) results of functional predictions based on metagenome composition (e.g. pathways or gene families).